src/engine/shared/demo.cpp

```diff
@@ -18,6 +18,18 @@ static const unsigned char gs_ActVersion = 4;
 static const int gs_LengthOffset = 152;
 static const int gs_NumMarkersOffset = 176;
 
+static inline void EncodeUnsigned(unsigned char *pOut, unsigned Value)
+{
+	pOut[0] = (Value>>24)&0xff;
+	pOut[1] = (Value>>16)&0xff;
+	pOut[2] = (Value>>8)&0xff;
+	pOut[3] = Value&0xff;
+}
+
+static inline unsigned DecodeUnsigned(const unsigned char *pIn)
```

Comment:
We have a function for that https://github.com/teeworlds/teeworlds/blob/master/src/base/system.c#L2615
Encoding could be moved there too.